extension-manifest-v2/src/classes/Common.js

```diff
@@ -84,7 +85,9 @@ common.load = async () => {
 
 	setPref('modules.anolysis.enabled', false);
 
-	if (!IS_ANDROID) {
+	if (!HAS_WASM) {
+		// Do NOT enable WTM reporting as e2e encryption is relatively CPU heavy
```

Comment:
e2e encryption is misleading, since it is about signing (or more precisely direct anonymous attestation).

Thinking about it, I would recommend to move the HAS_WASM prerequisite only around hpnv2+human-web. human-web-lite and hpn-lite don't use wasm.

Conceptually:

```
if (ANDROID) {
  		setPref('modules.human-web-lite.enabled', conf.enable_human_web);
		setPref('modules.hpn-lite.enabled', conf.enable_human_web);
} else if (HAS_WASM) {
	        setPref('modules.human-web.enabled', conf.enable_human_web);
		setPref('modules.hpnv2.enabled', conf.enable_human_web);
}
```